From 8a0d96eb13c97134e2587accab8509215a7db603 Mon Sep 17 00:00:00 2001
From: dann frazier <dann.frazier@chainguard.dev>
Date: Tue, 15 Sep 2026 11:33:32 -0600
Subject: [PATCH 08/18] Revert "tests: env: avoid false failure on FreeBSD"

This reverts commit 493ed58f218d4775ff209606ddf27f22adc83685.
NEWS left untouched.
---
 tests/env/env0-from.sh | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/tests/env/env0-from.sh b/tests/env/env0-from.sh
index 66d46de4c..9e54e7e47 100755
--- a/tests/env/env0-from.sh
+++ b/tests/env/env0-from.sh
@@ -93,8 +93,8 @@ case $host_os in
     sort all >out || framework_failure_
     compare exp-sorted out || fail=1
 
-    env -i --env0-from=base -u A C=operand D=new \
-      "$envexe" >exp || fail=1
+    env -i --env0-from=base \
+      "$envexe" -u A C=operand D=new >exp || fail=1
     sort exp >exp-sorted || framework_failure_
     env -i --env0-from=base \
       "$envexe" --env0-from=merge -u A C=operand D=new >all || fail=1
-- 
2.53.0

